pages/common/ncmpcpp.md

```diff
@@ -0,0 +1,19 @@
+# ncmpcpp
+
+> Ncmpcpp is a command line music player.
```

Comment:
Perhaps we could reword this such that the command name isn't mentioned in its description? Here's the description of the command from the apt package: https://hastebin.com/tamososaya

So perhaps we could do something like `A command line music player client for the Music Player Daemon?`?